### PR TITLE
BUGFIX: BB-1359 apply custom format for AM change

### PR DESCRIPTION
**Pull request by jkittler**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/183/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/215/ : FAILURE in 59s

File: src/DataLayer/converters/tests/MeasureConverter.spec.ts [hunk 1]
```diff
@@ -133,5 +133,12 @@ describe('convertMeasure', () => {
                 measures.buildArithmeticMeasure('am1', { operator: 'change' });
             expect(MeasureConverter.getFormat(arithmeticMeasure)).toEqual('#,##0.00%');
         });
+
+        it('should return custom format for change operation', () => {
+            const arithmeticMeasure =
```
Comment (tomassatinsky) [inline src/DataLayer/converters/tests/MeasureConverter.spec.ts]: I think preferred format would be
```
const arithmeticMeasure = measures.buildArithmeticMeasure(
    'am1', {operator: 'change'}, 'testAlias', 'customFormat'
);
```

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/184/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/216/ : FAILURE in 27s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/185/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/217/ : FAILURE in 24s

File: src/DataLayer/converters/tests/MeasureConverter.spec.ts (hunk 1 above)
Comment (jkittler) [reply src/DataLayer/converters/tests/MeasureConverter.spec.ts]: done

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/186/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/218/ : SUCCESS in 54s

**Pull request closed without merging**